src/main/java/core/basesyntax/dao/impl/UserDaoImpl.java

```diff
@@ -12,21 +16,69 @@ public UserDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public User create(User entity) {
-        return null;
+        Session session = null;
+        Transaction transaction = null;
+        try {
+            session = factory.openSession();
+            transaction = session.beginTransaction();
+            session.persist(entity);
+            transaction.commit();
+        } catch (HibernateException e) {
+            if (transaction != null) {
+                transaction.rollback();
+            }
+            throw new RuntimeException("Couldn't create the user in DB. User: " + entity, e);
+        } finally {
+            if (session != null) {
+                session.close();
+            }
+        }
+        return entity;
     }
 
     @Override
     public User get(Long id) {
-        return null;
+        User user = null;
+        try (Session session = factory.openSession()) {
+            user = session.get(User.class, id);
+        } catch (HibernateException e) {
+            throw new RuntimeException("Couldn't get the user from DB by id: " + id, e);
+        }
+        if (user == null) {
+            throw new RuntimeException("There is no user in DB by such id: " + id);
+        }
```

Comment:
Cool point, we can't return optional here so we should throw an exception! :fire: